src/test/java/StationTest.java

```diff
@@ -0,0 +1,142 @@
+import com.google.gson.Gson;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StationTest
```

Comment:
You don't need individual tests (all technically the same) to test your getters. This can all be done in one test.